src/last-changes/README.md

```
@@ -0,0 +1,116 @@
+## Options Reference
+
+| Option | Alias | Description | Default |
+|--------|-------|-------------|---------|
+| `--commits` | `-c` | Show changes from the last N commits | uncommitted |
+| `--verbose` | `-v` | Enable verbose logging | `false` |
+
+---
+
+## Output Example
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)